cmd/common/config.go

```diff
@@ -153,6 +159,11 @@ func NewConfigCmd(runtime runtime.Runtime, rootFlags *pflag.FlagSet) *cobra.Comm
 			if flag.Name == "config" {
 				return
 			}
+			if sv, ok := flag.Value.(pflag.SliceValue); ok {
+				defaultConfig.Set(flag.Name, sv.GetSlice())
+				return
+			}
```

Comment:
> Why do you need this code?

Just so the default value for slices/array flags as `[]` compared to `'[]'` (notice single quote). This will hint user to provide proper YAML list (handled by other code in this commit)

> Defvalue cannont contain a slice?

Unfortunately it is type `string` so it contains `'[]'`, which can be confusing!